src/components/Gradebook/StatusAlerts.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { StatusAlert } from '@edx/paragon';
+
+import { closeBanner } from '../../data/actions/grades';
+
+export class StatusAlerts extends React.Component {
+  get courseGradeFilterAlertDialogText() {
+    let dialogText = '';
+    if (!this.props.isMinCourseGradeFilterValid) {
+      dialogText += 'Minimum course grade value must be between 0 and 100. ';
+    }
+    if (!this.props.isMaxCourseGradeFilterValid) {
+      dialogText += 'Maximum course grade value must be between 0 and 100. ';
+    }
+    return dialogText;
+  }
+
+  render() {
+    return (
+      <div>
+        <StatusAlert
+          alertType="success"
+          dialog="The grade has been successfully edited. You may see a slight delay before updates appear in the Gradebook."
+          onClose={this.props.handleCloseSuccessBanner}
+          open={this.props.showSuccessBanner}
+        />
+        <StatusAlert
+          alertType="danger"
+          dialog={this.courseGradeFilterAlertDialogText}
+          dismissible={false}
+          open={
+            !this.props.isMinCourseGradeFilterValid
```

Comment:
If you put this check in a getter, you can do a single snapshot, and then a logic test on the method (rather than 3 snapshots)